group_vars/all

```diff
@@ -28,7 +28,11 @@ galaxy_extras_apt_package_state: present
 tool_dependency_dir: /home/{{ galaxy_user_name }}/tool_dependencies
 shed_tools_dir: "{{ galaxy_server_dir }}/../shed_tools"
 tool_data_dir: "{{ galaxy_server_dir  }}/tool-data"
-additional_files_list: []
+
+additional_files_list:
+    - { src: "extra-files/tool_sheds_conf.xml", dest: "{{ galaxy_config_dir }}" }
+    - { src: "extra-files/dependency_resolvers_conf.xml", dest: "{{ galaxy_config_dir }}/dependency_resolvers_conf.xml" }
```

Comment:
I wouldn't copy the resolver file, galaxy works fine without it, and we wouldn't automatically receive updates to the default resolver configuration. This is similar to datatypes, only copy and modify this file if there is a strong reason to do so.